### ISSUE TITLE
demo content is empty?

### ISSUE THREAD

**Issue by mikeveeckmans** [CaribThemes/grav-theme-tintamarre#5]
Hi, the demo contains empty pages? 
installed the pages from _demo to user/pages except for the background and the menu nothing shows up in the frontend...
Thanks.

**Comment by MATsxm** [comment]
hehehe! :wink: 
2 notes:
1- the Skeleton will be available next week at the latest... if the J! project gives me enough time for that :stuck_out_tongue_winking_eye: 
2- I tried this time (still looking for the best solution) to keep a maximum of content (except for the "about" page because I would like to try something new with 1 content and 2 layouts) in the `languages.yaml` file.
but I confess, I don't know why at this point, but sometimes, the `_demo` content seems hard to "push" on a previous install. I keep on testing & testing again ;)

**Comment by MATsxm** [comment]
Now that I'm curious, does the `_demo/pages` charged the way it is supposed to? or is it just because (except for the 'About') you found no content on `_demo` files?

**Comment by mikeveeckmans** [comment]
there was content in the _demo files , and the correct content is in the files when i open it with markdown editor but it doesn't show on the page , probably because of the skeleton...

**Comment by MATsxm** [comment]
nope, this isn't the way it supposed to work and I would say it's more a config prob even if I heard about the same issue for another theme. not that you are not able yet to install the theme from the backend (not yet on Grav repository) and will be soon.

The Skeleton (for a fresh install) is available here: https://github.com/CaribThemes/grav-skeleton-tintamarre/releases (not yet on Grav).

Anyway, if you want me to take a further look, ping me :wink:
